python/paddle/tensor/math.py

```diff
@@ -3371,6 +3371,153 @@ def cumsum(x, axis=None, dtype=None, name=None):
         return _cum_sum_(**kwargs)
 
 
+def cummax(x, axis=None, dtype='int64', name=None):
+    """
+    The cumulative max of the elements along a given axis.
+
+    Note:
+        The first element of the result is the same as the first element of the input.
+
+    Args:
+        x (Tensor): The input tensor needed to be cummaxed.
+        axis (int, optional): The dimension to accumulate along. -1 means the last dimension. The default (None) is to compute the cummax over the flattened array.
+        dtype (str, optional): The data type of the indices tensor, can be int32, int64. The default value is int64.
+        name (str, optional): Name for the operation (optional, default is None). For more information, please refer to :ref:`api_guide_Name`.
+
+    Returns:
+        out (Tensor), The result of cummax operation. The dtype of cummax result is same with input x.
+        indices (Tensor), The corresponding index results of cummax operation.
```

Comment:
![image](https://github.com/PaddlePaddle/Paddle/assets/70642955/0987a6cd-736e-4f9a-8594-52aa925ada3a)
没有换行成功
```suggestion
        out (Tensor), The result of cummax operation. The dtype of cummax result is same with input x.
        
        indices (Tensor), The corresponding index results of cummax operation.
```

Reply:
Done

Reply:
Done